.github/actions/init-blacksmith/action.yml

```diff
@@ -0,0 +1,138 @@
+name: Setup Action (Blacksmith)
```

Comment:
This action is separate from our existing `init` action due to its use of Blacksmith-specific actions, namely `useblacksmith/setup-node` and `useblacksmith/cache`.